index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+    <title>Portfolio: Setup and Mobile First</title>
+</head>
+<body>
+    <main class="header-background-mobile">
+        <header class="toolbar">
+            <nav>
+                <div class="mylogo">
+                    <a href="#">My Logo</a>
+                </div>
+                <div class="union">
+                    <img src="./images/Union.png" alt="Hamburguer Menu">
+                </div>
+            </nav>
+        </header>
+        <section class="headline">
+            <div class="primarytext">
+                <h1>I'm Federico Glad to see you!</h1>
+            </div>
+            <div class="bodytext">
+                <p>I'm a software developer! I can help you build a product , feature or 
+                    website Look through some of my work and experience! If you like what 
+                    you see and have a project you need coded, don't hestiate to contact me.
+                </p>
+            </div>
+            <div class="socialmedia">
+                <div id="supportingtext">
+                    <h5>LET'S CONNECT</h5>
+                </div>
+                <div>
+                    <ul class="socialmediaicons">
+                        <li><img src="./images/Vector.svg" alt="Twitter"></li>
+                        <li><img src="./images/Linkedinicon.svg" alt="Linkedin"></li>
+                        <li><img src="./images/Vector1.svg" alt="Mail"></li>
+                        <li><img src="./images/Vector2.svg" alt="GitHub"></li>
+                        <li><img src="./images/Vector3.svg" alt="Peace"></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="images">
+                <img id="elipse11" src="./images/Ellipse11.png" alt="">
+                <img id="elipse5" src="./images/Ellipse5.png" alt="">
+                <img id="rectangle7" src="./images/Rectangle7.png" alt="">
+                <img id="elipse3" src="./images/Ellipse3.png" alt="">
+                <img id="rectangle4" src="./images/Rectangle4.png" alt="">
+                <img id="elipse8" src="./images/Ellipse8.png" alt="">
+                <img id="elipse12" src="./images/Ellipse12.png" alt="">
+                <img id="elipse13" src="./images/Ellipse13.png" alt="">
+                <img id="elipse15" src="./images/Ellipse15.png" alt="">
+                <img id="elipse4" src="./images/Ellipse4.png" alt="">
+                <img id="rectangle5" src="./images/Rectangle5.png" alt="">
+            </div>
```

Comment:
- [ ] Kindly make sure the decorations for the section are always **background images**, not `<img>` tags) i.e you are required to use `CSS background-image` property.